src/main/java/business/models/RequestProperties.java

```diff
@@ -373,4 +375,17 @@ public void setReseachNumber(String reseachNumber) {
         this.reseachNumber = reseachNumber;
     }
 
+    public boolean getGermlineMutation() {
+        if (germlineMutation == null){
+            germlineMutation = false;
+        }
+        return germlineMutation;
+    }
+
+    public void setGermlineMutation(Boolean germlineMutation) {
+        if (germlineMutation == null){
+            germlineMutation = false;
```

Comment:
This means that you lose the distinction between 'Yes', 'No', and nothing selected, right?